hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/hapi/ContractUpdateSuite.java

```diff
@@ -512,6 +515,35 @@ final Stream<DynamicTest> cannotUpdateMaxAutomaticAssociations() {
                 .then(contractUpdate(CONTRACT).newMaxAutomaticAssociations(20).hasKnownStatus(NOT_SUPPORTED));
     }
 
+    @LeakyHapiTest(PROPERTY_OVERRIDES)
+    final Stream<DynamicTest> tryContractUpdateWithMaxAutoAssociations() {
+        return propertyPreservingHapiSpec("tryContractUpdateWithMaxAutoAssociations")
+                .preserving("contracts.allowAutoAssociations")
+                .given(
+                        overriding("contracts.allowAutoAssociations", TRUE_VALUE),
```

Comment:
Consider preserving the `ledger.maxAutoAssociations `as 5000, if this value is changed it will brake the test